### PR TITLE
feat: support custom prop

### PR DESCRIPTION
**Pull request by DSergiu**

### CODE REVIEW

**Review by wontem — APPROVED** [review]
Looks good

File: CHANGES.md
Comment (SyedSeemahabib) [inline CHANGES.md:1]: Duplicate of 0x8C6a39174b2618da2e9CdFdF75E0131C5b090D31#### ###

Comment (SyedSeemahabib) [reply CHANGES.md:1]: Ok

File: README.md
Comment (SyedSeemahabib) [inline README.md:1]: README.md

**Pull request merged**